.github/ISSUE_TEMPLATE/proposal.yml

```diff
@@ -0,0 +1,40 @@
+name: Proposal
+description: Proposal for feature to include in Interop 2023
+labels: proposal
+body:
+- type: markdown
+  attributes:
+    value: |
+      Thanks for taking the time to make a proposal for Interop 2023!
+
+      Please see the [Interop 2023 README](https://github.com/web-platform-tests/interop/blob/main/2023/README.md) for the timeline and what to expect from the process.
+- type: textarea
+  attributes:
+    label: Description
+    description: Brief description of the feature. Try to be clear about what parts of the feature and included and not.
+- type: textarea
+  attributes:
+    label: Specification
+    description: |
+      Specification for the feature. The feature should be defined by a standards-track specification from the following organizations:
+
+      - [IETF](https://www.ietf.org/) (Standard Track)
+      - [Khronos Group](https://www.khronos.org/)
+      - [TC39](https://tc39.es/) (Stage 2+)
+      - [W3C](https://www.w3.org/) (Recommendation Track)
+      - [WHATWG](https://whatwg.org/)
+
+      Other specifications will be considered on a case-by-case basis.
+- type: textarea
+  attributes:
+    label: Tests
+    description: Tests for the feature in web-platform-tests, on the [wpt.fyi](https://wpt.fyi/) dashboard. Other test suites will be considered on a case-by-case basis.
```

Comment:
I think we can be stronger here: features without web-platform-tests and where there's no short-term plan to have web platform tests can't be part of a normal focus area. They could possibly be part of an "investigate" item where the scope might include adding web-platform tests or other infrastructure needed to generate test-based scores.

Reply:
I'd suggest we accept WPT and Test262, and don't say we'll consider other test suites. For Test262 we should ask for links to test262.report I guess until we have results on wpt.fyi.

Reply:
```suggestion
    description: Tests for the feature in web-platform-tests on [wpt.fyi](https://wpt.fyi/), or in Test262 on [test262.report](https://test262.report/).
```